src/css/vendor/_variables_uswds.scss

```diff
@@ -6,115 +6,58 @@
 // Removing the !default from $em-base so we are not inheriting that
 // value from Bourbon.
 
-@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,700');
-@import "../../../node_modules/uswds/src/stylesheets/lib/addons/font-stacks";
-
-$em-base:             10px;
-$base-font-size:      1.7rem !default;
-$small-font-size:     1.4rem !default;
-$lead-font-size:      2rem !default;
-$title-font-size:     5.2rem !default;
-$h1-font-size:        4rem !default;
-$h2-font-size:        3.6rem !default;
-$h3-font-size:        2.6rem !default;
-$h4-font-size:        2rem !default;
-$h5-font-size:        1.7rem !default;
-$h6-font-size:        1.3rem !default;
-$base-line-height:    1.4 !default;
-$heading-line-height: 1.3 !default;
-$lead-line-height:    2 !default;
-
-$font-sans:           'Montserrat', $helvetica !default;
-$font-serif:          'Merriweather', $georgia !default;
+// @import "../../../node_modules/uswds/src/stylesheets/lib/addons/font-stacks";
```

Comment:
delete instead of comment out?

Reply:
kinda same as above.  I wanted to show what overrides uswds made available -- and uncommented them as I needed them, so I wouldn't have to keep referencing their github. I can def delete the ones we don't end up needing as we wrap up the v2 implementation.